### ISSUE TITLE
Why do I get the Loss Nan in imdb_cnn.py?

### ISSUE THREAD

**Issue by NobodyWHU** [fchollet/keras#3167]
I used Theano for the keras.

When I ran the example code imdb_cnn.py, I got `nan` for loss after a few seconds.
I changed the .theanorc.txt files as follows:

> [global]
> openmp=False
> allow_gc=False
> optimizer = fast_run
> floatX = float32
> device = cpu
> [blas]
> ldflags= 
> [gcc]
> cxxflags = -ID:\Anaconda2\MinGW\include 

So how do I solve this problem?
thanks.

**Comment by ekerazha** [comment]
I've the same issue with keras 1.0.5, while keras 1.0.4 works fine... @fchollet
